src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SupplementaryEvidenceTest.java

```diff
@@ -28,6 +29,7 @@
 
 @SpringBootTest
 @ActiveProfiles("nosb") // no servicebus queue handler registration
+@Disabled
```

Comment:
Why are these tests disabled?

Reply:
Sorry, I have added WIP to the title. This is just temporarily for debugging CreateCase test